### PR TITLE
Added dummy intake route (Julianne + Juthika)

### PR DESCRIPTION
**Pull request by juthikahoque**
## Notion ticket link

[Add dummy create /intakes route](https://www.notion.so/uwblueprintexecs/Add-dummy-create-intakes-route-26e861b0610b4a6f9f47f9d2e2f5473a)

## Implementation description
* created functions with dummy content for a dummy intake route

## Steps to test
1. run this command: `docker exec -it cas_py_backend /bin/bash -c "pip install -e . && pytest"` to make sure that all tests pass (which it should because it is hardcoded to pass)
2. run this command: `curl -X POST http://localhost:5000/intake` to make sure that the object returned is `{"status": "ok"}`

## What should reviewers focus on?
* nothing in particular, but maybe focus on the new files added

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by juthikahoque** [comment]
i tried squashing some commits together and in the process it added more commits instead - it should still be fine tho!

File: backend/python/app/rest/intake_routes.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import json
+
+from flask import Blueprint, current_app, jsonify, request
+
+from ..middlewares.auth import require_authorization_by_role
+
+# from ..middlewares.validate import validate_request
+# from ..services.implementations.intake_service import IntakeService
+
+# define instance of EntityService
```
Comment (jennchenn) [inline backend/python/app/rest/intake_routes.py:10]: nit: delete

File: backend/python/app/rest/intake_routes.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import json
+
+from flask import Blueprint, current_app, jsonify, request
+
+from ..middlewares.auth import require_authorization_by_role
+
+# from ..middlewares.validate import validate_request
```
Comment (jennchenn) [inline backend/python/app/rest/intake_routes.py:7]: i think you can safely uncomment these

File: backend/python/app/rest/intake_routes.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# from ..services.implementations.intake_service import IntakeService
+
+# define instance of EntityService
+# intake_service = IntakeService(current_app.logger, file_storage_service)
+
+# defines a shared URL prefix for all routes
+blueprint = Blueprint("intake", __name__, url_prefix="/intake")
```
Comment (jennchenn) [inline backend/python/app/rest/intake_routes.py:14]: nit:
```suggestion
blueprint = Blueprint("intake", __name__, url_prefix="/intakes")
```

File: backend/python/app/services/implementations/intake_service.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from ..interfaces.file_storage_service import IFileStorageService
+from ..interfaces.intake_service import IIntakeService
+
+
+class IntakeService(IIntakeService):
+    def __init__(self, logger, file_storage_service: IFileStorageService):
```
Comment (jennchenn) [inline backend/python/app/services/implementations/intake_service.py:6]:
```suggestion
    def __init__(self, logger):
```

you can also initialize the logger like in the `user_service`
https://github.com/uwblueprint/childrens-aid-society/blob/0c7838db69596a1b3d45f7d70f0be8bd6492ee8b/backend/python/app/services/implementations/user_service.py#L9-L21

File: backend/python/app/services/interfaces/intake_service.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+
+
+class IIntakeService(ABC):
+    """
```
Comment (jennchenn) [inline backend/python/app/services/interfaces/intake_service.py:5]: nit: go through and rename `entity`->`intake`

**Review by jennchenn — CHANGES_REQUESTED** [review]
woo first PR 🌟 a few comments

i like using fixup for small changes:
`git commit --fixup <hash of commit your fixing>`
you can also try:
`git rebase -i HEAD~13` or `git rebase -i main` but it's fine for now (we do squash commits when merging into main anyway so it doesn't really matter how many commits there are, just for readability for reviewers)

File: backend/python/app/rest/intake_routes.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+# define POST endpoint for creating an intake
+@blueprint.route("/", methods=["POST"], strict_slashes=False)
+# @require_authorization_by_role({"User", "Admin"})
+# @validate_request("IntakeDTO")
+def create_intake():
+    return (
```
Comment (jennchenn) [inline backend/python/app/rest/intake_routes.py:21]: also feel free to change the body to just be a `pass`

File: backend/python/app/services/interfaces/intake_service.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    A class to handle CRUD functionality for intakes
+    """
+
+    @abstractmethod
+    def create_intake(self, intake):
+        """Create a new Entity object
```
Comment (jennchenn) [inline backend/python/app/services/interfaces/intake_service.py:11]: nit:
```suggestion
        """Create a new Intake object
```

File: backend/python/app/rest/intake_routes.py [hunk 8]
```diff
@@ -0,0 +1,16 @@
+import json
+
+from flask import Blueprint, current_app, jsonify, request
```
Comment (jennchenn) [inline backend/python/app/rest/intake_routes.py:3]: nit: remove unused imports
```suggestion
from flask import Blueprint
```

File: backend/python/app/rest/intake_routes.py [hunk 9]
```diff
@@ -0,0 +1,16 @@
+import json
+
```
Comment (jennchenn) [inline backend/python/app/rest/intake_routes.py:2]: nit: remove the first two lines in this file
```suggestion
```

File: backend/python/app/rest/intake_routes.py [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+from ..middlewares.auth import require_authorization_by_role
+from ..middlewares.validate import validate_request
+
+# defines a shared URL prefix for all routes
+blueprint = Blueprint("intake", __name__, url_prefix="/intakes")
+
+# define POST endpoint for creating an intake
+@blueprint.route("/", methods=["POST"], strict_slashes=False)
+# @require_authorization_by_role({"User", "Admin"})
+# @validate_request("IntakeDTO")
```
Comment (jennchenn) [inline backend/python/app/rest/intake_routes.py:14]: nit: uncomment these two lines
```suggestion
@require_authorization_by_role({"User", "Admin"})
@validate_request("IntakeDTO")
```

**Review by jennchenn — COMMENTED** [review]
nice work! once you resolve these small things should be good to merge!

**Review by jennchenn — APPROVED** [review]
lgtm!! nice work 🚀

**Pull request merged**